mmdeploy/codebase/mmrotate/models/test_mixins.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Since there is no `test_mixins.py` in `mmrotate/roi_heads`, you can just name this file `rotated_bbox_head.py` or `bbox_heads`.

Reply:
OK